docs/integration/kubernetes.md

```diff
@@ -823,3 +823,22 @@ Also, please make sure that your agent is [up to date](https://sematext.com/docs
 
 ### Why we need hostNetwork access and how to turn it off if desired
 Check out [our page about hostnetwork](https://sematext.com/docs/agents/sematext-agent/kubernetes/hostnetwork/).
+
+### Setting TLS Certificate Paths
+While monitoring Kubernetes master components, the Sematext agent will automatically retrieve certificates from the host machine. 
```

Comment:
Sematext agent ==> Sematext Agent (we treat it as a name elsewhere, I think?)

Reply:
Modified, also checked if there were more cases like this, didn't found places in which we don't use Sematext Agent